packages/buckram/assets/styles/components/_colors.scss

```diff
@@ -167,7 +167,7 @@ h1.front-matter-title {
 }
 
 // PART TITLE COLOR
-h3.part-number {
+p.part-number {
```

Comment:
Wouldn't it be better if we removed element selectors altogether and just targeted elements with the .part-number class?

Reply:
Yes, we discussed that on Friday I wasn't sure if the `.part-number` exists in other contexts